src/02-common-principles.md

```diff
@@ -585,6 +582,11 @@ Corollaries:
     across multiple data files, it is RECOMMENDED to store metadata in this
     way, rather than duplicating that metadata content across multiple metadata files.
 
+1.  Where multiple applicable JSON files are loaded as per rule 5.2, key-values can
+    only be overwritten by files lower in the filesystem hierarchy; the absence of
+    a key-value in a later file does not imply the "unsetting" of that field
+    (and indeed there is no notion of such).
```

Comment:
isn't there a word missing there?

Reply:
I tend to only use "such" in this way when the object to which it refers is unambiguous and can therefore be implicit. But it might be clumsy just from trying to preserve previous phrasing. See if you prefer 2590d0a.

Reply:
I like the rewording 👍

Reply:
Same here